src/main/java/thePackmaster/vfx/discopack/SpotlightFXtest.java

```diff
@@ -0,0 +1,57 @@
+package thePackmaster.vfx.discopack;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.Interpolation;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.helpers.ImageMaster;
+import com.megacrit.cardcrawl.vfx.AbstractGameEffect;
+import thePackmaster.SpireAnniversary5Mod;
+
+import static thePackmaster.SpireAnniversary5Mod.modID;
+
+public class SpotlightFXtest extends AbstractGameEffect {

```

Comment:
Might be nice to rename this, test seems kind of weird here since it is being used